app/graphql/mutations/account/grant_site_permission.rb

```diff
@@ -0,0 +1,29 @@
+class Mutations::Account::GrantSitePermission < Mutations::Base
+  argument :account, ID,
+    required: true,
+    description: 'Who to grant permissions to'
+
+  argument :permission, Types::Enum::SitePermission,
+    required: true,
+    description: 'The permission to grant to this user'
+
```

Comment:
If we want to follow the way other mutations are implemented, we should have this inside an input.

Reply:
What do you mean by that? If you mean wrapping it in an input type, that's really not helpful for a simple mutation?